HomeWork11/Task1_2/Printable.java

```diff
@@ -0,0 +1,28 @@
+package HomeWork11.Task1_2;
+
+public interface Printable {
+    void print();
+
+    /*При этом варианте у меня не получилось получить ответ
+     от методов Book.printBooks(printables) и Magazine.printMagazines(printables);
+        static Printable create(String type, String title) {
+        type = type.toLowerCase();
+        if ("book".equals(type)) {
+            return () -> System.out.println("Book: " + title);
+        } else if ("magazine".equals(type)) {
+            return () -> System.out.println("Magazine: " + title);
+        } else {
+            return () -> System.out.println("Unknown type");
+        }*/
+
+        static Printable create(String type, String title) {
```

Comment:
Object creation should not be at this place. Main logic of the interface is to outline behaviour. Also, since Java 8 we can add some static methods but it just the same as in abstract class(some utility methods that cover some default behaviour, not the object creating)